README.md

```diff
@@ -4,21 +4,44 @@
 
 > This JavaScript capstone project is built on the [TVMAZE API](https://static.tvmaze.com/apidoc/), an external API. The cards display the movies information that was retrieved from the API. The user has the option to like and comment on the movies cards.
 
+![](src/assets/cinema.JPG)
+
+![](src/assets/mobil.JPG)
+
 ## Built With
 
-- HTML, CSS, JavaScript, and API
+- HTML, CSS, JavaScript, and Tv-Maze API
 
 ## Live Demo (if available)
 
-[Live Demo Link](https://livedemo.com)
+[Live Demo Link](https://natanima.github.io/Javascript-capstone-project/)
 
 ## Getting Started
 
-**This is an example of how you may give instructions on setting up your project locally.**
-**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
 
 To get a local copy up and running follow these simple example steps.
 
+
+```
+git clone https://github.com/NatanimA/Javascript-capstone-project.git
+```
+
+```
+npm install
+```
+
+```
+npm run build
+```
+
+Lastly
+
+```
+npm start
+```
+
```

Comment:
- Hey, I could start your app but how do I run the test in your application? Please, kindly add the command to run the test and also to make your readme more professional giving the user extra features about the app.

Reply:
![s](https://user-images.githubusercontent.com/66979348/197012487-a10ea567-aa02-46a2-b83c-a647c4751d9c.JPG)

I clearly stated how you can test it @zieeco